lib/js-yaml/type/js/function.js

```diff
@@ -11,8 +11,8 @@ var esprima;
 //
 try {
   // workaround to exclude package from browserify list.
-  var _require = require;
-  esprima = _require('esprima');
+  // eslint-disable-next-line no-eval
+  esprima = eval('require')('esprima');
```

Comment:
I'd like to avoid such `eval` use.

Reply:
I know its not perfect.. I tried many other things but apparently, webpack is quite intelligent. Another way would be to use the `IgnorePlugin` webpack provides, which is less obvious. If you want it to be so, I would be down to write a note in the readme that this should be considered to keep bundle size low. It was more of an accident that I found it in our bundle and I'd like to save others the time for finding out :)